qiita_pet/handlers/api_proxy/tests/test_studies.py

```diff
@@ -232,6 +238,105 @@ def test_study_prep_get_req(self):
         for i in range(4, 0, -1):
             qdb.artifact.Artifact(i).visibility = "private"
 
+    def test_study_prep_get_req_failed_EBI(self):
+        temp_dir = mkdtemp()
+        self._clean_up_files.append(temp_dir)
+        user_email = 'test@foo.bar'
+
+        # creating a (A) new study, (B) sample info, (C) prep without EBI
+        # values
+
+        # (A)
+        # ignoring warnings generated when adding templates
+        simplefilter("ignore")
```

Comment:
Already discussed with @antgonza offline but noticing here for others in the thread. The problem is that this line is ignoring the warning and this applies to all future tests